### PR TITLE
OpenEBS blog excerpt and tags separator

### PR DESCRIPTION
**Pull request by satyapriyamishra222**
Add OpenEBS blog excerpt and tags separator

### CODE REVIEW

**Review by AVRahul — APPROVED** [review]
lgtm

File: website/src/blogs/how-are-tiktok-flipkart-kubesphere-and-others-using-openebs-for-local-volumes.md [hunk 1]
```diff
@@ -2,8 +2,9 @@
 title: How are TikTok, Flipkart, KubeSphere, and others using OpenEBS for Local Volumes
 author: Kiran Mova
 author_info: Contributor and Maintainer OpenEBS projects. Chief Architect MayaData. Kiran leads overall architecture & is responsible for architecting, solution design & customer adoption of OpenEBS.
-tags: solutions tutorials chaosengineering
+tags: Localpv, Openebs, Flipkart, TikTok, Kubernetes, Mayastor, Mayadata
```
Comment (Pallavi-PH) [inline website/src/blogs/how-are-tiktok-flipkart-kubesphere-and-others-using-openebs-for-local-volumes.md:5]:
```suggestion
tags: Localpv, Openebs, Flipkart, TikTok, Kubernetes, Mayastor, MayaData
```

File: website/src/blogs/openebs-ndm-goto-solution-for-managing-kubernetes-local-storage.md [hunk 2]
```diff
@@ -4,6 +4,7 @@ author: Akhil Mohan
 author_info: Software Engineer @ MayaData, working on Cloud Native Tech.
 tags: openebs
 date: 13-01-2021
+excerpt: Read about OpEBS NDM, the go-to solution for managing Kubernetes Local Storage.
```
Comment (Pallavi-PH) [inline website/src/blogs/openebs-ndm-goto-solution-for-managing-kubernetes-local-storage.md:7]:
```suggestion
excerpt: Read about OpenEBS NDM, the go-to solution for managing Kubernetes Local Storage.
```

File: website/src/blogs/how-are-tiktok-flipkart-kubesphere-and-others-using-openebs-for-local-volumes.md (hunk 1 above)
Comment (IsAmrish) [reply website/src/blogs/how-are-tiktok-flipkart-kubesphere-and-others-using-openebs-for-local-volumes.md:5]: also can we change --> 

Openebs ==>  OpenEBS

File: website/src/blogs/repeatable-openebs-mayastor-deployments-and-benchmarks.md [hunk 3]
```diff
@@ -2,8 +2,9 @@
 title: Repeatable OpenEBS Mayastor deployments and benchmarks
 author: OPENEBS
 author_info: 
-tags: tutorials openebs
+tags: Mayastor,Openebs
```
Comment (IsAmrish) [inline website/src/blogs/repeatable-openebs-mayastor-deployments-and-benchmarks.md:5]: Openebs => OpenEBS

File: website/src/components/BlogsSlider/index.tsx [hunk 4]
```diff
@@ -82,10 +83,7 @@ const BlogsSlider: React.FC<BlogsSliderProps> = ({ recommendedBlogs }) => {
                     <span>
                       <ReactMarkdown
                         children={
-                          elm.content
-                            .substring(0, 200)
-                            .replace(/[\n]/g, ". ")
-                            .replace(/[^a-zA-Z ]/g, "") + "..."
+                          getContentPreview(elm.excerpt)
```
Comment (Pallavi-PH) [inline website/src/components/BlogsSlider/index.tsx:86]: I think we should add word limit here to maintain the same content length as per the design, even if `excerpt` length provided is lengthy

File: website/src/blogs/how-are-tiktok-flipkart-kubesphere-and-others-using-openebs-for-local-volumes.md (hunk 1 above)
Comment (satyapriyamishra222) [reply website/src/blogs/how-are-tiktok-flipkart-kubesphere-and-others-using-openebs-for-local-volumes.md]: Done

File: website/src/blogs/openebs-ndm-goto-solution-for-managing-kubernetes-local-storage.md (hunk 2 above)
Comment (satyapriyamishra222) [reply website/src/blogs/openebs-ndm-goto-solution-for-managing-kubernetes-local-storage.md]: Done

File: website/src/blogs/repeatable-openebs-mayastor-deployments-and-benchmarks.md (hunk 3 above)
Comment (satyapriyamishra222) [reply website/src/blogs/repeatable-openebs-mayastor-deployments-and-benchmarks.md]: Done

**Review by IsAmrish — APPROVED** [review]
lgtm

**Review by Pallavi-PH — APPROVED** [review]
Lgtm

**Pull request merged**